.github/workflows/ci.yml

```diff
@@ -86,6 +86,11 @@ jobs:
         with:
           command: update
           args: -p log --precise 0.4.14
+      - name: Downgrade bytemuck
```

Comment:
These downgrade steps really do make our buildscripts quite ugly... Maybe we just want to force an explicit version of bytemuck in Smithay?

Although I believe a dependency has this set.